admin-dev/themes/new-theme/scss/components/_hint_box.scss

```diff
@@ -1,8 +0,0 @@
-.hint-box {
```

Comment:
Used inside

`src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/bootstrap_4_layout.html.twig`

Maybe you changed it later and I don't see it, but I'm leaving the comment anyway.

Reply:
The style of this `.hint-box` is like an alert with a thicker left border, which to me makes no sense because we are trying to harmonize the BO. Therefore, it has no impact for me if the hint box displays as an alert, because the Twig file `bootstrap_4_layout.html.twig` uses this markup: `<div class="alert alert-info">`, so it will render as an info alert. That's why I have removed this part.